prov/gni/src/gnix_cm.c

```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright (c) 2015-2016 Cray Inc.  All rights reserved.
- * Copyright (c) 2015 Los Alamos National Security, LLC. All rights reserved.
+ * Copyright (c) 2015-2017 Cray Inc.  All rights reserved.
+ * Copyright (c) 2015-2017 Los Alamos National Security, LLC.
+ *                         All rights reserved.
```

Comment:
Since Cray hasn't technically made any changes to this document, I don't know if I'd change the dates for Cray. Ask Howard or Sung. It honestly shouldn't matter much at this point since this is a team project for us.